File: Zestawy Zadań/C++/4 - Tablice/Zestaw 2 zadania/4.2.3 - Tablica z n liczbami calkowitymi.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int n;
    cin >> n;
    while (n < 1 || n > 201)
    {
        cout << "Musisz wprowadzic liczbe w zakresie (1 <= n <= 200): ";
        cin >> n;
    }
    vector<double> tab(n);
    for (int i = 0; i < n; i++)
    {
        cin >> tab[i];
    }
    for (int i = n-1; i >= 0; i--)
    {
        cout << tab[i] << " ";
    }
}

File: Zestawy Zadań/Python/4 - Tablice/Zestaw 2 zadania/4.2.3 - Tablica z n liczbami calkowitymi.py
n = int(input("Podaj n w zakresie (1 <= n <= 200): "))

while n < 1 or n > 200:
    n = int(input("Podaj n w zakresie (1 <= n <= 200): "))

tab = [0] * n

for i in range(0, n):
    tab[i] = int(input("Podaj liczbe do tablicy: "))

print(tab)
